### DISCUSSION TITLE
actual results not beeing sent

### DISCUSSION THREAD

**Discussion by Ludoplh-van-Ceulen** [preda/gpuowl#288] [Q&A] [answered]
Hi, I am using GPUowl in GIMPS project. It works fine and calculations are done on my GPU/VGA. However, sending actual reports of reserved exponent calculations, does not work. 

I have these two dirs ".\proof" and ".\proof-tmp" folders in dir root where GPUowl is situated (these ".\proof" and ".\proof-tmp" are empty).

Also I have directiory ".\150587083" which is my reserved exponent and the are dirs:
".\150587083\proof" with cca 18 MiB files
and also in ".\150587083" are cca 18 MiB files

I have GpuOwl VERSION v7.5-10-g61a08e2
and
Primenet.exe (with built in Primenet.py and all execution stuff) in dir root where GPUowl is situated.
Primenet version is 2.1.

Nowadays I have 4,5% of calculation behind me. I am trying to use Primenet for sending actual reports to GIMPS web.

Using cmdlet ".\primenet.exe --username -[MyUserID] --status" it tells me:
INFO: M150587083, PRP, Finish cannot be estimated  
(BUT WHY ???)
decimal places and chance of yield a Mersenne prime is cool, but why Primenet cannot do ETA?

Also:
.\primenet.exe --upload-proofs
.\primenet.exe --upload-proofs --username -[MyUserID] --upload-proofs
.\primenet.exe --upload-proofs --username -[MyUserID] --upload-proofs --dir=150587083
.\primenet.exe --upload-proofs --username -[MyUserID] --password [MyUserPWD] --upload-proofs --dir=150587083
.\primenet.exe --upload-proofs --username -[MyUserID] --password [MyUserPWD] --upload-proofs --dir=150587083 --checkin=1
.\primenet.exe --upload-proofs --username -[MyUserID] --password [MyUserPWD] --upload-proofs --dir=150587083 --checkin=1
.... absolutely nothing works, the actual reports are NOT sent to GIMPS web :(

Is anybody using Primenet besides GPUowl to send actual results? Can you give a help?

**Comment by tdulcet** [comment, marked as answer]
Sorry to hear that you are having problems. Note that the PrimeNet program and GpuOwl are separate projects.

The `--upload-proofs` option can only upload proof files if they exist. You mentioned that your `.\proof` directory is empty, so this option will not do anything. You would need to wait for the PRP test to finish and then the PrimeNet program should automatically upload the resulting proof file if it is running.

The `--dir` option is only for users that have multiple instances of GpuOwl on their system, in which case it should be provided once for each instance with the path to the root directory for GpuOwl.

Providing your PrimeNet password is deprecated and disables many features, including assignment progress reporting. If you open your `local.ini` file and delete the `password=` line, the program should return to normal.

If the `--status` option is still unable to determine the assignment progress, it means either you never provided the `--gpuowl` option or the program was unable to find the `gpuowl.log` file. You could add the `--debug` option for more details about the problem. Most of the options (including the `--gpuowl` option) are saved in the config file and thus only need to be provided once.

I would recommend reading [the documentation](https://download.mersenne.ca/primenet.py) for more information about how to correctly use the PrimeNet program with GpuOwl.

**Reply by Ludoplh-van-Ceulen** [reply]
Hello, many thanks for warp-speedy answer :)
Yes I noticed those are different projects and I had a little soul that somebody will answer Primenet problems in GPUowl space.

--upload-proofs ... yep thats why I tried copying files from 
".\150587083\proof" to the ".\proof" and also to ".\proof-tmp"
and also from ".\150587083" to ".\" but no effect

--password [MyUserPWD] ... I am aware of the primenet PSW deprecation, but I was trying everything, I deleted the line from primenet config file local.ini, no effect

.\primenet.exe --username -[MyUserID] --gpuowl --status
.\primenet.exe --username -[MyUserID] --status
.\primenet.exe --gpuowl --status
.\primenet.exe --status
(nothing works for INFO: M150587083, PRP, Finish cannot be estimated)

I have to look deeper into primenet page. Thanks.

**Reply by tdulcet** [reply]
In the future, the fastest and best way to get support is on the [Mersenne forum](https://www.mersenneforum.org/).

GpuOwl does not generate the proof file until the very end of the PRP test. You mentioned that your test was only 4% complete, so it still has a long way to go before the proof file would be generated...

Regarding the assignment progress reporting not working, the PrimeNet program should be in the same directory as GpuOwl, which should include (among other things) the `gpuowl.log` and `results.txt` files, as well as the `proof` directory. If that does not work, please share the output from: `.\primenet.exe --status --debug` and the last 20 lines of the `gpuowl.log` file.

**Comment by Ludoplh-van-Ceulen** [comment]
This Q can be considered as answered. Simply its like this:
- GpuOwl does not generate the proof file until the very end of the PRP test.
- so there is nothing to send for AutoPrimeNet/PrimeNet

I have tested, once the calculation is 100% done, proof file generated and results.txt updated, AutoPrimeNet/PrimeNet sends the results smoothly.
(& there si also possibility manual uploading via GIMPS page)
